### PR TITLE
[zh] sync news announcing-1.19-eol

### PR DESCRIPTION
**Pull request by jiecloud**
## Description
[zh] sync news announcing-1.19-eol

## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

**Comment by windsonsea** [comment]
Check the lint error:

```
./zh/news/support/announcing-1.19-eol/index.md:16: MD047 File should end with a single newline character
```

File: content/zh/news/support/announcing-1.19-eol/index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+根据 Istio 的[支持政策](/zh/docs/releases/supported-releases#support-policy)，
+针对 1.19 这类次要版本的支持将持续到 N+2 次要版本（在本例中为 1.21）发布后的六周。
+[Istio 1.21 于 2024 年 03 月 13 日发布](/zh/news/releases/1.21.x/announcing-1.21/)，
+所以对 1.19 的支持于 2024 年 04 月 24 日结束。
+
+届时，我们将停止向后修复 1.19 中的安全问题和严重错误，
+因此我们鼓励您升级到最新发布的 Istio ({{<istio_release_name>}})。
+如果您不这样做，您可能会陷入必须在短时间内进行重大升级才能获得关键修复的境地。
+
+我们关心您和您的集群，所以请善待自己的集群并尽快升级。
```
Comment (windsonsea) [inline content/zh/news/support/announcing-1.19-eol/index.md:17]:
```suggestion
我们关心您和您的集群，所以请善待自己的集群并尽快升级。

```
Add a blank line at bottom, to fix the lint error

File: content/zh/news/support/announcing-1.19-eol/index.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+title: 对 Istio 1.19 的支持于 2024 年 04 月 24 日结束
+subtitle: 支持公告
+description: Istio 1.19 生命周期即将结束的公告。
+publishdate: 2024-03-27
+---
+
+根据 Istio 的[支持政策](/zh/docs/releases/supported-releases#support-policy)，
+针对 1.19 这类次要版本的支持将持续到 N+2 次要版本（在本例中为 1.21）发布后的六周。
+[Istio 1.21 于 2024 年 03 月 13 日发布](/zh/news/releases/1.21.x/announcing-1.21/)，
+所以对 1.19 的支持于 2024 年 04 月 24 日结束。
```
Comment (wilsonwu) [inline content/zh/news/support/announcing-1.19-eol/index.md:11]:
```suggestion
[Istio 1.21 于 2024 年 3 月 13 日发布](/zh/news/releases/1.21.x/announcing-1.21/)，
所以对 1.19 的支持于 2024 年 4 月 24 日结束。
```

**Review by wilsonwu — COMMENTED** [review]
Suggest to keep same format of date with other announcements.

File: content/zh/news/support/announcing-1.19-eol/index.md (hunk 1 above)
Comment (jiecloud) [reply content/zh/news/support/announcing-1.19-eol/index.md]: Hi @windsonsea 
Thanks for your comment. I have added a blank line at the bottom of the file to fix the lint error.
Please let me know if you have any other questions.

File: content/zh/news/support/announcing-1.19-eol/index.md (hunk 2 above)
Comment (jiecloud) [reply content/zh/news/support/announcing-1.19-eol/index.md]: Hi @wilsonwu ,
Thanks for your comment. I have changed the data format.
Please let me know if you have any other questions.

File: content/zh/news/support/announcing-1.19-eol/index.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+title: 对 Istio 1.19 的支持于 2024 年 04 月 24 日结束
```
Comment (wilsonwu) [inline content/zh/news/support/announcing-1.19-eol/index.md:2]:
```suggestion
title: 对 Istio 1.19 的支持于 2024 年 4 月 24 日结束
```

File: content/zh/news/support/announcing-1.19-eol/index.md (hunk 1 above)
Comment (windsonsea) [reply content/zh/news/support/announcing-1.19-eol/index.md]: If you have changed by following suggestions, you can click the `Resolve conversation` button, so every reviewer will easily know the progress

**Review by wilsonwu — APPROVED** [review]
LGTM

**Pull request merged**